content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,89 @@
+ ([Issue #22762](https://github.com/istio/istio/issues/22762))
+
+- **Enabled** Prometheus [metrics merging](/docs/ops/integrations/prometheus/#option-1-metrics-merging) by default.
+ ([Issue #21366](https://github.com/istio/istio/issues/21366))
+
+- **Fixed** Prometheus [metrics merging](/docs/ops/integrations/prometheus/#option-1-metrics-merging) to not drop Envoy metrics during application failures.
+ ([Issue #22825](https://github.com/istio/istio/issues/22825))
+
+- **Removed** the `pilot_xds_eds_instances` and `pilot_xds_eds_all_locality_endpoints` Istiod metrics, which were not
+accurate.
+ ([Issue #25154](https://github.com/istio/istio/issues/25154))
+
+## Installation
+
+- **Added** RPM packages for running the Istio sidecar on a VM to the release.
+ ([Issue #9117](https://github.com/istio/istio/issues/9117))
+
+- **Fixed** an issue preventing `NodePort` services from being used as the `registryServiceName` in `meshNetworks`.
+
+- **Improved** gateway deployments to run as non-root by default.
+ ([Issue #23379](https://github.com/istio/istio/issues/23379))
+
+- **Upgraded** the CRD and Webhook versions to `v1`.
+ ([Issue #18771](https://github.com/istio/istio/issues/18771)),([Issue #18838](https://github.com/istio/istio/issues/18838))
+
+## istioctl
+
+- **Added** Allow proxy-status for non Kubernetes workloads with `--file`
+
+- **Added** a configuration file to hold istioctl default flags. Its default location (`$HOME/.istioctl/config.yaml`) can be changed using the environment variable `ISTIOCONFIG`. The new command `istioctl experimental config list` shows the default flags.
+ ([Issue #23868](https://github.com/istio/istio/issues/23868))
+
+- **Added** `--revision` flag to `istioctl operator init` and `istioctl operator remove` commands to support multiple control plane upgrade.
+ ([Issue #23479](https://github.com/istio/istio/issues/23479))
+
+- **Added** `istioctl analyze` now warns if deprecated mixer resources are present
+ ([Issue #24471](https://github.com/istio/istio/issues/24471))
+
+- **Added** `istioctl x uninstall` command to uninstall Istio control plane.
+ ([Issue #24360](https://github.com/istio/istio/issues/24360))
+
+-  **Added** `istioctl analyze` now warns if DestinationRule is not using CaCertificates to validate server identity.
+
+- **Improved** `istioctl validate` to check for unknown fields in resources.
+ ([Issue #24861](https://github.com/istio/istio/issues/24861))
+
+- **Removed** `istioctl manifest apply`. The simpler `install` command replaces manifest apply.
+ ([Issue #25737](https://github.com/istio/istio/issues/25737))
+
+## Documentation changes
+
+- **Added** support for auto generating release notes based off of a file present in the `releasenotes/notes` directory.
```

Comment:
Does user care about this?

Reply:
I thought the same. This is an internal tooling improvement, not a user facing change.